### PR TITLE
ref(metrics): renames operation to aggregation

### PR DESCRIPTION
**Pull request by obostjancic**
- renames `op` to `aggregation` in metric types to distinguish it form `span.op`

### CODE REVIEW

File: static/app/utils/metrics/dashboardImport.tsx [hunk 1]
```diff
@@ -315,9 +315,11 @@ export class WidgetParser {
     const appliedFunctionMatch = str.match(/\.(\w+)\(\)/);
     const appliedFunction = appliedFunctionMatch ? appliedFunctionMatch[1] : undefined;
 
-    if (!op) {
-      this.errors.push(`widget.request.query - could not parse op: ${str}, assuming sum`);
-      op = 'sum';
+    if (!aggregation) {
+      this.errors.push(
+        `widget.request.query - could not parse aggregation: ${str}, assuming sum`
+      );
```
Comment (priscilawebdev) [inline static/app/utils/metrics/dashboardImport.tsx:321]: are we displaying these errors somewhere to clients? shouldn't we use the translation function here?

**Comment by priscilawebdev** [comment]
since metrics like the average and p90 are categorized under "operations," it makes sense to rename them as "aggregates." 

The code looks good to me, but it is hard to review due to so many needed changes. I will leave it to those more familiar with this part of the code to approve.

File: static/app/utils/metrics/dashboardImport.tsx (hunk 1 above)
Comment (obostjancic) [reply static/app/utils/metrics/dashboardImport.tsx:321]: yes, but they are visible only to sentry staff since this is an internal feature

Comment (priscilawebdev) [reply static/app/utils/metrics/dashboardImport.tsx:321]: ok we just have to remember to update it when the feature is available to clients

**Review by vgrozdanic — APPROVED** [review]
Let's 🐑 this

**Pull request merged**